packages/web-api/src/admin/routes.ts

```diff
@@ -28,16 +28,16 @@ router.post(
   passport.authenticate('jwt', { session: false }),
   assertUserIsAdminMiddleware,
   async (req, res: Response<DataSpecificationUpdateRequestResponse>) => {
-    console.log('Received request to create data specification update job');
+    console.debug('Received request to create data specification update job');
```

Comment:
generally, would be nice to do this kind of change in separate PR so there aren't so many files in a feature PR